themes/docsy/assets/scss/_custom.scss

```diff
@@ -903,6 +903,69 @@ body.tv-episode {
   margin-right: 10px;
 }
 
+.checklist-item {
+  padding: 2rem 2rem 1rem;
+  margin: 1.5rem 0;
```

Comment:
To my eye this feels like too much whitespace, but maybe we can get a real designer like Rachel or Alex to take a look...

Reply:
Have changed these, they look a little tighter now.